### PR TITLE
[#160949] change ethernet port to string

### PR DESCRIPTION
**Pull request by jossim**
# Release Notes

This changes the `ethernet_port_number` on Relays from an integer to a string

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -55,6 +55,34 @@
     t.index ["affiliate_id"], name: "index_accounts_on_affiliate_id"
   end
 
+  create_table "active_storage_attachments", charset: "utf8mb3", force: :cascade do |t|
```
Comment (giladshanan) [inline db/schema.rb:58]: This seems unrelated

File: db/schema.rb [hunk 2]
```diff
@@ -185,7 +213,7 @@
     t.boolean "is_active", null: false
     t.integer "created_by", null: false
     t.datetime "created_at", null: false
-    t.integer "revenue_account", null: false
+    t.string "revenue_account", null: false
```
Comment (giladshanan) [inline db/schema.rb:216]: this too...

**Review by giladshanan — CHANGES_REQUESTED** [review]
Looks like your schema file has some unrelated changes in it.  What happens to existing data when this migration runs?

File: db/schema.rb (hunk 1 above)
Comment (jossim) [inline db/schema.rb:58]: When I ran `rails db:migrate`, these active storage and affiliate changes happened, though it didn't seem there was a specific migration that was run to make these changes. So I couldn't figure out how to prevent them and also migrate the relays change

Comment (giladshanan) [reply db/schema.rb:58]: @jossim try running `db:rollback` then check the schema, repeat until the active storage and affiliate changes are removed.  at that point you may have to get a little creative - perhaps deleting or commenting out the active storage migration - then run `db:migrate` again.

**Review by giladshanan — APPROVED** [review]
